### PR TITLE
SG-30515 - breakdown2 app TK release documentation updates

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/9b8c92afd6762d29ab6b4fbec16ae754ed6247e4/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5016f8523cecf77df682f839e187011037f9cc80/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/6d888a0acb0092a387a67f944d83c3f6a8af01ed/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/6dcc9628dbc7a398ddb61f68d52d7eb37f7c96db/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/2f4a98962b84df01f62330d73a0fdcd51e4db604/index.html)

File: docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-breakdown.md [hunk 1]
```diff
@@ -7,19 +7,36 @@ lang: en
...
+![Overview of the breakdown2 app](../images/apps/07_scenebreakdown1.png)
 
-## Scanning the Scene
+Please refer to the following documentation for the latest breakdown app:
 
-When the breakdown app scans the scene for references, it will call a special scanning hook. The scanning hook will return a list of nodes with file corresponding paths. For each reference it finds, it returns the file path, and tank will look at the file path and first check if it recognizes it as a publish, and if it does, see if there is a more recent version available.
+- [Breakdown App User Documentation](https://help.autodesk.com/view/SGSUB/ENU/?guid=VRED_ShotGrid_for_Automotive_Design_VRED_ShotGrid_Worflows_Scene_Breakdown_in_VRED_html)
+- [Breakdown App Developer API Documentation](https://developer.shotgridsoftware.com/tk-multi-breakdown2/)
+- [Release Notes](https://github.com/shotgunsoftware/tk-multi-breakdown2/wiki/Release-Notes)
+
```
Comment (rob-aitchison) [inline docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-breakdown.md:24]: I would love to see a horizontal line to divide the page, other than that looks great - thanks!

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/4323ea0739cc1fcb8215c5fc6ec7f3566008cf58/index.html)

File: docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-breakdown.md (hunk 1 above)
Comment (rob-aitchison) [reply docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-breakdown.md:24]: I like it - do you agree @shaynacohen ? Thanks for adding in in there!

Comment (shaynacohen) [reply docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-breakdown.md:24]: Yes! It looks slick. Thanks for the suggestion! I was not expecting it to work the first time (I was experimenting). Let me know when this is good to publish, and I can merge!

**Review by rob-aitchison — APPROVED** [review]
Looks great - merge away! :)

**Pull request merged**